ui/de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterWindowEventHandler.java

```diff
@@ -199,7 +200,7 @@ public void onRequestSimpleUpload(RequestSimpleUploadEvent event) {
                                        diskResourceServiceProvider.get(),
                                        eventBus,
                                        diskResourceUtil,
-                                       UriUtils.fromTrustedString(clientConstants.fileUploadServlet()),
+                                       UriUtils.fromTrustedString(clientConstants.ontologyUploadServlet()),
```

Comment:
Nope, another oversight from testing.  This should be removed now from the latest commit.